src/ecephys_etl/modules/vbn_create_stimulus_table/create_stim_table.py

```diff
@@ -883,9 +890,9 @@ def compute_vbn_block_frame_times(
     # Now deal with leftover vsyncs that occur after the last diode transition
     # Just take the global frame duration for these
     leftover_vsyncs_start_ind = (
-        len(partitioned_vsync_times)
-        - np.mod(len(partitioned_vsync_times), num_vsyncs_per_diode_toggle)
+        (len(partitioned_photodiode_times) - 1) * num_vsyncs_per_diode_toggle
```

Comment:
I'm not sure that photodiode times should be mixed with expected `num_vsyncs_per_diode_toggle` at this point in the code.

The photodiode signal can be thought of as the indicator for when a frame change actually shows up on the display screen *after* a vsync signal is sent... Can you double check with Corbett that what you have here is actually desired?

Reply:
This is the recommended solution from http://confluence.corp.alleninstitute.org/x/7Yl-Bg

Reply:
Are we sure that the solution in that Confluence page wasn't just a rough-and-ready proof of concept? It seems like, if what we really want are all vsyncs that occur after the last photodiode edge, we should be comparing the values in `partitioned_vsync_times` to the values in `partitioned_photodiode_times`.

Unless those two arrays correspond to different clocks and cannot sensibly be compared (this is one of those cases where having more explicit documentation in this code explaining what each input and output physically is would be really helpful).

Reply:
I think it is correct as is. Every photodiode_time corresponds to 60 vsyncs (which is the num_vsyncs_per_diode_toggle). This is not the same as finding all vsyncs after the last photodiode time.

Reply:
I've emailed Corbett about this just to make sure...

Reply:
Corbett says this is fine.